### ISSUE TITLE
rinstall plugin bug when setup SN on sles12.2 ppc64le

### ISSUE THREAD

**Issue by xuweibj** [xcat2/xcat-core#3466]
sles12.2 
ppc64le 

```
Jul 14 13:47:47 c910f03c11k12 xcat[10652]:  xCAT: Allowing rinstall to c910f03c11k13 osimage=sles12.2-ppc64le-install-service for root from localhost
Jul 14 13:47:51 c910f03c11k12 xcat[10653]:  Can't use string ("internal error: qemu unexpectedl"...) as an ARRAY ref while "strict refs" in use at /opt/xcat/lib/perl/xCAT/Utils.pm line 1453.
Jul 14 13:47:51 c910f03c11k12 xcat[10653]:  xcatd: rinstall plugin bug, pid 10653, process description: 'xcatd SSL: rinstall to c910f03c11k13 for root@localhost: rinstall instance: kvm instance' with error 'Died at /opt/xcat/sbin/xcatd line 1999.#012' while trying to fulfill request for the following nodes: c910f03c11k13
--
```

**Comment by xuweibj** [comment]
```
Jul 18 13:33:29 c910f03c11k12 xcat[8841]:  xCAT: Allowing rinstall to c910f03c11k13 osimage=sles12.2-ppc64le-install-service for root from localhost
Jul 18 13:33:33 c910f03c11k12 xcat[8842]:  Can't use string ("internal error: qemu unexpectedl"...) as an ARRAY ref while "strict refs" in use at /opt/xcat/lib/perl/xCAT/Utils.pm line 1453.
Jul 18 13:33:33 c910f03c11k12 xcat[8842]:  xcatd: rinstall plugin bug, pid 8842, process description: 'xcatd SSL: rinstall to c910f03c11k13 for root@localhost: rinstall instance: kvm instance' with error 'Died at /opt/xcat/sbin/xcatd line 1999.#012' while trying to fulfill request for the following nodes: c910f03c11k13
```

**Comment by whowutwut** [comment]
@xuweibj  Is this the same issue as #3495 ??

**Comment by xuweibj** [comment]
@whowutwut This is not the same. This occured when installed SN.

**Comment by immarvin** [comment]
this issue occurs when deploy VM

**Comment by immarvin** [comment]
fixed in https://github.com/xcat2/xcat-core/pull/3507

**Comment by immarvin** [comment]
since this is a block issue in daily autotest, change the priority to "high"

**Comment by immarvin** [comment]
The verification is covered by daily autotest:
````
RUN:rinstall c910f03c11k13 osimage=sles12.2-ppc64le-install-service [Wed Jul 19 11:55:29 2017]
ElapsedTime:5 sec
RETURN rc = 1
OUTPUT:
Error: Failed to run 'rpower' against the following nodes: c910f03c11k13
Provision node(s): c910f03c11k13
c910f03c11k13: internal error: qemu unexpectedly closed the monitor: 2017-07-19T15:55:34.608881Z qemu-kvm: Failed to allocate KVM HPT of order 26 (try smaller maxmem?): Cannot allocate memory
CHECK:rc == 0   [Failed]
````

the internal exception of libvirt is successfully captured and reported by ``rinstall``

close this ticket
